src/TimePicker.elm

```diff
@@ -151,6 +151,21 @@ defaultTime =
     }
```

Comment:
I think it would be a good idea to remove this binding and inline it in the one place it's used as a filter initial value.

Reply:
done.